concrete/controllers/single_page/dashboard/reports/security.php

```diff
@@ -29,6 +29,7 @@ public function __construct(Page $page, Service $siteService, SiteValidatorManag
     public function view()
     {
         $result = new \ArrayIterator();
+        $this->manager->setEnvironment($this->app->environment());
```

Comment:
This shouldn't map to the site environment, this should be either `Environment::DEVELOPMENT` or `Environment::PRODUCTION`. Originally I envisioned this being a switch in the dashboard to turn your site to production mode.